app/internal/import_file.py

```diff
@@ -45,26 +57,54 @@ def is_file_exist(file: str) -> bool:
     return Path(file).is_file()
 
 
+def change_string_to_date(date: str) -> Union[datetime.datetime, bool]:
+    try:
+        if re.search(r":", date):
+            date1 = datetime.datetime.strptime(date, DATE_FORMAT2)
+        else:
+            date1 = datetime.datetime.strptime(date, DATE_FORMAT)
+    except ValueError:
```

Comment:
What is the purpose of the except here?

Reply:
if the string is not a valid date `datetime.strptime` will fail to transfer the string to date and we want to catch it.